string speech_command
string speech_command_feedback